externals/tecio/examples/brickpartitioned/brickpartitioned.vcxproj

```diff
@@ -136,7 +136,7 @@
     </Midl>
     <ClCompile>
       <Optimization>Disabled</Optimization>
-      <AdditionalIncludeDirectories>..\..\..\..\include;C:\Program Files\Microsoft MPI\Inc;C:\Program Files (x86)\Microsoft SDKs\MPI\Include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>..\..\..\..\include;..\..\teciompisrc;C:\Program Files\Microsoft MPI\Inc;C:\Program Files (x86)\Microsoft SDKs\MPI\Include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>

```

Comment:
Can the .vcxproj files be removed from version control?

Reply:
Yep, I'll remove those and related Visual Studio-only files.